docs/basics/vue.md

```diff
@@ -98,11 +98,13 @@ importing its default export and using it [as a plugin](https://v3.vuejs.org/gui
 import { createApp } from 'vue';
 import Root from './App.vue';
 import urql from '@urql/vue';
+import { cacheExchange, fetchExchange } from '@urql/core';
```

Comment:
These can also both be imported from `@urql/vue`, so it's not necessary to add `@urql/core` to your dependencies.